portfolio/src/main/webapp/script.js

```diff
@@ -26,6 +26,9 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
-
-
-
+function addMessage() {
```

Comment:
nit:
1. Add blank line before this method.
2. Add a description of what this method does.

Reply:
solved